src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -462,27 +459,27 @@ function formatNumberBoxes() {
 }
 
 var updateVisibilityOfNumEntitiesBox = function() {
-    var questionNum = $(this).prop('id').split('-')[1];
-    questionNum = questionNum || '';
-
-    var value = $(this).val();
-
-    formatNumberBox(value, questionNum);
+    var questionId = getQuestionId($(this));
```

Comment:
Might be better to leave this as `questionNum`, and the associated function name changed; `Id` is more likely to be taken as the id of the entity in the backend, instead of the index of the question (which is the case here)